saleor/graphql/attribute/utils.py

```diff
@@ -303,22 +311,33 @@ def save(cls, instance: T_INSTANCE, cleaned_input: T_INPUT_MAP):
         :param cleaned_input: the cleaned user input (refer to clean_attributes)
         """
         pre_save_methods_mapping = {
-            AttributeInputType.FILE: cls._pre_save_file_value,
-            AttributeInputType.REFERENCE: cls._pre_save_reference_values,
-            AttributeInputType.RICH_TEXT: cls._pre_save_rich_text_values,
-            AttributeInputType.PLAIN_TEXT: cls._pre_save_plain_text_values,
-            AttributeInputType.NUMERIC: cls._pre_save_numeric_values,
             AttributeInputType.BOOLEAN: cls._pre_save_boolean_values,
             AttributeInputType.DATE: cls._pre_save_date_time_values,
             AttributeInputType.DATE_TIME: cls._pre_save_date_time_values,
+            AttributeInputType.DROPDOWN: cls._pre_save_dropdown_value,
+            AttributeInputType.FILE: cls._pre_save_file_value,
+            AttributeInputType.NUMERIC: cls._pre_save_numeric_values,
+            AttributeInputType.MULTISELECT: cls._pre_save_multiselect_values,
+            AttributeInputType.PLAIN_TEXT: cls._pre_save_plain_text_values,
+            AttributeInputType.REFERENCE: cls._pre_save_reference_values,
+            AttributeInputType.RICH_TEXT: cls._pre_save_rich_text_values,
```

Comment:
It's getting bigger. It seems like it could be extracted as the Mixin is getting really big. Just wanted to mention it to consider changing it in a separate PR.